src/Source/EntwinePointTileSource.js

```diff
@@ -0,0 +1,79 @@
+import proj4 from 'proj4';
+import LASParser from 'Parser/LASParser';
+import Fetcher from 'Provider/Fetcher';
+import Source from 'Source/Source';
+
+class EntwinePointTileSource extends Source {
+    /**
+     * @constructor
+     *
+     * @param {Object} config - The configuration
+     * @param {Object} config.parserOptions - The options to pass to the parser.
+     * See the documentation of {@link LASParser} or {@link PotreeBinParser} to
+     * see the available options, given the format of your points.
+     */
+    constructor(config) {
+        super(config);
+
+        // https://entwine.io/entwine-point-tile.html#ept-json
+        const json = Fetcher.json(`${this.url}/ept.json`, this.networkOptions)
+            .then(metadata => Object.assign(this, metadata));
+
+        this.sources = {};
+        this.sourcesMetadata = {};
+        const list = Fetcher.json(`${this.url}/ept-sources/list.json`, this.networkOptions)
+            .then((list) => {
+                list.forEach((obj) => {
+                    this.sources[obj.id] = obj;
+                    if (obj.url && !this.sourcesMetadata[obj.url]) {
+                        this.sourcesMetadata[obj.url] = {};
+                    }
+                });
+
+                // eslint-disable-next-line
+                for (const s in this.sourcesMetadata) {
+                    Fetcher.json(`${this.url}/ept-sources/${s}`, this.networkOptions).then((source) => {
+                        this.sourcesMetadata[s] = source;
+                    });
+                }
+            });
```

Comment:
Could you remove this part and add unsupported bin format comment?

Reply:
Yes